src/module/keyboard-layouts/fr.js

```diff
@@ -0,0 +1,41 @@
+//SPDX-FileCopyrightText: 2021 Max VS (Maximus#6001)
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2021 Max VS (Maximus#6001)
```

Reply:
Hey, I made the requested changes in an additional commit